packages/pci-sdk-iframe/src/pages/app/components/Debugger.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { PropsWithChildren, useEffect, useState } from 'react';
+
+type IDebuggerProps = any;
+
+export default function Debugger(props: IDebuggerProps) {
+	const [responses, setResponses] = useState<IResponse[]>([]);
+	const [mountDate] = useState<Date>(() => new Date());
+
+	useEffect(() => {
+		const { fetch: origFetch } = window;
+		(window as Window).fetch = async function fetchWithDebug(input: RequestInfo, init?: RequestInit) {
+			const response = await origFetch(input, init);
+			const res = await response.clone();
+			const serverResponse = await res.json();
+			const body = init?.body;
+
+			const newResponse: IResponse = {
+				body,
+				headers: init?.headers,
+				serverResponse,
+				status: res.status,
+				url: res.url,
+			};
+
+			setResponses((s: IResponse[]) => [...s, newResponse]);
+
+			return response;
+		};
+	}, []);
+
+	return (
+		<aside
+			style={{
+				position: 'fixed',
+				background: 'white',
```

Comment:
💅  I guess we don't need this background rule because we have the backgroundColor set below.
```suggestion
```